agent/stats/task_linux.go

```diff
@@ -223,9 +220,11 @@ func (taskStat *StatsTask) getAWSVPCNetworkStats(deviceList []string, containerP
 						return
 					}
```

Comment:
Just curious what will be the behaviour when err==nil and deviceList = []?

Reply:
It might be possible that in the first run, devicelist=[] and err==nil because network devices hasn't yet attached to the docker container, in that case networkstats will be empty. But because we are checking if deviceList is empty in every run, it will again query for the device list until it finds a device, which has to be there for task to make network requests.